File: app/src/main/java/com/olehmesh/randomusers_task/database/DatabaseManager.java
package com.olehmesh.randomusers_task.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.olehmesh.randomusers_task.models.EntityData;

@Database(entities = {EntityData.class}, version = 1, exportSchema = false)

public abstract class DatabaseManager extends RoomDatabase {

    public abstract DaoMethods daoMethods();

}

File: app/src/main/java/com/olehmesh/randomusers/repository/database/DatabaseManager.kt
package com.olehmesh.randomusers.repository.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.olehmesh.randomusers.repository.database.dao.DaoCurrentDate
import com.olehmesh.randomusers.repository.database.dao.DaoUserAndDate
import com.olehmesh.randomusers.repository.database.dao.DaoUserEntity
import com.olehmesh.randomusers.repository.database.entity.DateLatLng
import com.olehmesh.randomusers.repository.database.entity.UserEntity

@Database(entities = [UserEntity::class, DateLatLng::class], version = 1, exportSchema = false)
abstract class DatabaseManager : RoomDatabase() {

    abstract fun daoUserInfo(): DaoUserEntity
    abstract fun daoCurrentDate(): DaoCurrentDate
    abstract fun daoUserAndDate(): DaoUserAndDate

}